Uncaught Error: Cannot find module '@babel/runtime-corejs2/helpers/interopRequireWildcard'

**Description:**
Hi there,
I installed Hydrogen on Atom using the command line (apm install hydrogen) as it throws an error directly from the package manager. After installation I open Atom to run some test code (eg, print("hello")). Even before I try to "shift-Enter", the error pops up. Unable to run any code with Hydrogen "shift-enter", running the file as a script works fine though.
Spend a bit of time looking around ( I rarely write help posts), and the closest I found was the same type of error message from meteor: [Dates from April 2018]
https://forums.meteor.com/t/solved-cannot-find-module-babel-runtime-helpers-builtin-objectspread-after-update-meteor-to-1-6-1-1/43034

The problem and solution there was:

"The meteor-babel npm package has been updated to version 7.0.0-beta.42, which may require updating any custom Babel plugins you’ve enabled in a .babelrc file, and/or running the following command to update @babel/runtime
meteor npm install @babel/runtime@latest"

I wonder if it's the same issue with Hydrogen, and if there is a fix for it or if I'm doing something wrong.

Help :)

Thanks

PS: In troubleshooting guide, it says to add debug messages by ticking enable debug message.
I don't have that tick in the settings (picture attached).
![image](https://user-images.githubusercontent.com/4509592/45332076-dc9e4980-b5b1-11e8-944c-8ab0d1607593.png)

**Steps to Reproduce:**
(Hard to know since it happens when I open Atom after installation)
1. Install Hydrogen with command line
2. Open Atom
3. Error will pop-up

**Versions:**

**Atom**: 1.30.0 x64
**Electron**: 2.0.5
**OS**: Microsoft Windows 10 Enterprise
**python**: 3.6

**Logs:**
### Stack Trace

Uncaught Error: Cannot find module '@babel/runtime-corejs2/helpers/interopRequireWildcard'

```
At module.js:545

Error: Cannot find module '@babel/runtime-corejs2/helpers/interopRequireWildcard'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (~/AppData/Local/atom/app-1.30.0/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (~/AppData/Local/atom/app-1.30.0/resources/app/src/module-cache.js:354:58)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (~/AppData/Local/atom/app-1.30.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/Hydrogen/node_modules/@nteract/transform-vdom/lib/index.js:3:31)
    at /packages/Hydrogen/node_modules/@nteract/transform-vdom/lib/index.js:78:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.30.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.30.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.30.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/Hydrogen/node_modules/@nteract/transforms/lib/index.js:43:22)
    at /packages/Hydrogen/node_modules/@nteract/transforms/lib/index.js:110:3)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.30.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.30.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (~/AppData/Local/atom/app-1.30.0/resources/app/src/native-compile-cache.js:66:33)
    at /packages/Hydrogen/lib/store/output.js:3:52)
    at /packages/Hydrogen/lib/store/output.js:65:33)
    at Module.get_Module._compile (~/AppData/Local/atom/app-1.30.0/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (~/AppData/Local/atom/app-1.30.0/resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
```

### Commands

```
     -0:43 virtualenv:select (atom-workspace.workspace.scrollbars-visible-always.theme-atom-material-syntax-dark.theme-atom-dark-ui)
     -0:39.8.0 hydrogen:run-and-move-down (input.hidden-input)
```

### Non-Core Packages

```
atom-file-icons 0.6.0 
atom-material-syntax-dark 1.0.0 
atom-python-virtualenv 1.0.2 
atom-shortcuts 0.0.2 
autocomplete-python 1.10.11 
busy-signal 1.4.3 
Hydrogen 2.5.1 
intentions 1.1.5 
kite 0.104.0 
linter 2.2.0 
linter-flake8 2.3.0 
linter-ui-default 1.7.1 
minimap 4.29.8 
platformio-ide-terminal 2.8.4 
python-autopep8 0.1.3 
script 3.18.1 
```

--- a/package.json
+++ b/package.json
@@ -45,9 +45,13 @@
     "atom": ">=1.20.0 <2.0.0"
   },
   "lint-staged": {
-    "{lib,spec}/**/*.js": ["prettier --write", "git add"]
+    "{lib,spec}/**/*.js": [
+      "prettier --write",
+      "git add"
+    ]
   },
   "dependencies": {
+    "@babel/runtime-corejs2": "^7.0.0",
     "@jupyterlab/services": "^0.52.0",
     "@nteract/commutable": "^3.0.4",
     "@nteract/display-area": "^3.1.7",

